src/lib/y2network/connection_config/base.rb

```diff
@@ -70,6 +70,8 @@ class Base
       attr_accessor :firewall_zone
       # @return [String] interface's hostname
       attr_accessor :hostname
+      # @return [Boolean] set to true if dhcp from this interfae sets machine hostname
```

Comment:
```suggestion
      # @return [Boolean] set to true if dhcp from this interface sets machine hostname
```

Reply:
And not sure. I think we should permit also nil. File configuration overrides global. 

If set globally to set the hostname by dhcp but not per interface we should not write anything at all, or in other case we will override the global configuration.

If I get it correctly we will always set it so, it will be always be written in the ifcfg-file.